final/background/spoofax.tex

```diff
@@ -27,33 +27,34 @@ \section{The Spoofax Language Workbench}
 specification consists of the following conceptual steps:
 
 \begin{enumerate}
-\item \hyperref[sec:syntax-def]{Syntax Definition}: The first step defines what textual
+\item \hyperref[ssec:syntax-def]{Syntax Definition}: The first step defines what textual
 representations of a program are syntactically valid. A parser
 provides an implementation of this definition, by mapping a textual
 representation of a program to an abstract syntax tree (AST)
 representation. In Spoofax, the syntax is declared with a domain
 specific language (DSL) called SDF.
-\item \hyperref[sec:static-analysis]{Static Semantics}: The AST then goes through static analysis (type
+\item \hyperref[ssec:static-analysis]{Static Semantics}: The AST then goes through static analysis (type
 checking, name binding and variable scoping), to test if the
 program is well-formed. Static semantics describe the rules for the
 static analysis step. Spoofax provides two DSLs for the
-specification of this step: the TS type specification language and
+specification of this step: the TS Type Specification language and
```

Comment:
Reading this it seems to me like the user has a choice on whether he uses NABL or TS for static analysis, but these languages are not mutually exclusive and each have their own task (as is properly explained further on).